pkg/ccl/backupccl/create_scheduled_backup.go

```diff
@@ -632,6 +634,23 @@ func emitSchedule(
 	return nil
 }
 
+// returns true if a schedule with the same label already exists
+func checkScheduleAlreadyExists(
+	ctx context.Context, p sql.PlanHookState, scheduleLabel string,
+) (bool, error) {
+
+	// the last
+	row, err := p.ExecCfg().InternalExecutor.QueryRowEx(ctx, "check_sched",
+		p.ExtendedEvalContext().Txn, sessiondata.InternalExecutorOverride{User: security.RootUserName()},
+		fmt.Sprintf("SELECT COUNT(schedule_name) FROM %s WHERE schedule_name = '%s'",
+			scheduledjobs.ProdJobSchedulerEnv.ScheduledJobsTableName(), scheduleLabel))
+
+	if err != nil {
+		return false, err
+	}
+	return int64(tree.MustBeDInt(row[0])) != 0, err
```

Comment:
lets return `nil` instead of err since we have already handled the error above.

Reply:
done